components/nexus-security/pom.xml

```diff
@@ -45,23 +45,17 @@
     </dependency>
 
     <dependency>
-      <groupId>javax.enterprise</groupId>
-      <artifactId>cdi-api</artifactId>
-    </dependency>
-
-    <dependency>
-      <groupId>org.eclipse.sisu</groupId>
-      <artifactId>org.eclipse.sisu.inject</artifactId>
-    </dependency>
-
-    <dependency>
-      <groupId>org.sonatype.sisu</groupId>
-      <artifactId>sisu-guice</artifactId>
+      <groupId>org.sonatype.sisu.goodies</groupId>
+      <artifactId>goodies-eventbus</artifactId>
     </dependency>
 
+    <!--
+    NOTE: Here only for javax.enterprise.inject.Typed, consider removing use and dropping dependency?
```

Comment:
Ya, will likely do that in later changes but didn't want to risk if anything was dependent on this in this pull.  There was at least one that @Type() to a sub-interface wasn't sure how it would resolve so I left this note to remind to tidy it up.